scipy/stats/_unuran/unuran_wrapper.pyx.templ

```diff
@@ -603,9 +618,15 @@ cdef class TransformedDensityRejection(Method):
         Set factor for relative size of the guide table for indexed search.
         It must be greater than or equal to 0. When set to 0, then sequential
         search is used. Default is 2.
-    random_state : int, Generator, RandomState, optional
-        A NumPy random number generator or seed for the underlying NumPy random
-        number generator. Default is ``None`` in which case a random seed is used.
+    random_state : {None, int, `numpy.random.Generator`,
+                        `numpy.random.RandomState`}, optional
+
```

Comment:
You still probably still want something about how this determines the underlying uniform RNG.

Reply:
> You still probably still want something about how this determines the underlying uniform RNG. (here and below)

Done. Here, below, and in the `set_random_state` method.